frontend/src/screens/study-landing.tsx

```diff
@@ -18,20 +18,59 @@ import {
     Text,
     Title,
 } from '@mantine/core';
+import { IconCheck } from '@tabler/icons-react'
 import Markdown from 'react-markdown'
 import { useLearningPathStudies } from './learner/studies';
 import { CompactStudyCard } from '../components/study/compact-study-card';
+import { notifications } from '@mantine/notifications';
+
+let notificationShownThisSession = false;
```

Comment:
I dont think we should be doing it this way - we don't want to be modifying a variable outside of the component state.